src/graph/dg.rs

```diff
@@ -374,71 +374,191 @@ pub fn resolve_all_paths_only_of_references(
     log::info!("found {} bindings", bindings);
 }
 
-pub fn resolve_all_paths_manual_extension(
+// pub fn resolve_all_paths_manual_extension(
+//     explorer: &mut SynchroExplorer,
+//     stack_graph: &StackGraph
+// ) {
```

Comment:
This one is using `Path` so it didn't work. Doing this for `PartialPath` would not be very different. But ideally, the algorithms in stack graph would be good enough to do this, and you wouldn't need to do this yourself. (That's why I'm doing some performance debugging myself here and see if we can improve the library.)